share/governance.md

```diff
@@ -0,0 +1,78 @@
+
+
+Users
+-----
+
+
+Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements. Users should be encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+* advocating for use of the project
+* informing developers of project strengths and weaknesses from a new user’s perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* writing documentation and tutorials
+* filing bug reports and feature requests
+* participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a VoxPupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. Expectations are that contributors will submit patches that are small at first and will only grow larger once the contributor has built confidence in the quality of their patches.
+
+*How to become one*: Submit a pull request to a VoxPupuli project
+
+
+Contributors
+------------
+
+
+Contributors are contributors who have shown wide dedication to the VoxPupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators and users. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
```

Comment:
This was in the template... but I agree with it. In OpenStack we nominate and approve publicly, and what we've seen after years of doing that is no one is actually willing to downvote including someone publicly. On the infrastructure side we moved to doing it more privately, and I feel like the analysis of someones contributions to the project is more two-sided and honest.

I not against doing it publicly, but I think it will be overall harder on the community to do it that way. In general we try to praise people in public, and give criticism in private. Having a public mailing list where people discuss the good and the bad of your contributions to a project is going to be emotionally challenging. It's also difficult for the people reviewing your contributions to be honest in that situation. If a private conversation is had, then the response can be agreed upon. Either an announcement that a new committer has joined our ranks, or a private conversation between one or two committers and the candidate focused on where we'd like to see improvement.